interface-impl-amf/src/main/java/org/mule/amf/impl/model/AMFImpl.java

```diff
@@ -220,6 +224,24 @@ public String dumpAmf() {
     }
   }
 
+  public void writeAMFModel(OutputStream outputStream) {
+    try {
+      OutputStreamWriter writer = new OutputStreamWriter(outputStream, Charset.forName("UTF-8"));
+      new AmfGraphRenderer()
+          .generateToBuilder(consoleModel.get(),
+                             new RenderOptions()
+                                 .withoutSourceMaps()
+                                 .withoutPrettyPrint()
+                                 .withCompactUris(),
+                             new JsonOutputBuilder<>(writer, false,
+                                                     org.mulesoft.common.io.Output$.MODULE$.outputWriter()))
```

Comment:
I think you can use just `Output.outputWriter()`

Reply:
how ??

Reply:
`import org.mulesoft.common.io.Output;`
`new JsonOutputBuilder<>(writer, false, Output.outputWriter())`